gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/editor/QuickEditor.kt

```diff
@@ -0,0 +1,83 @@
+package com.gravatar.quickeditor.ui.editor
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Surface
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.gravatar.quickeditor.ui.avatarpicker.AvatarPicker
+import com.gravatar.quickeditor.ui.components.EmailLabel
+import com.gravatar.quickeditor.ui.components.ProfileCard
+import com.gravatar.quickeditor.ui.components.QEPageDefault
+import com.gravatar.ui.GravatarTheme
+
+@Composable
+internal fun QuickEditor(
+    gravatarQuickEditorParams: GravatarQuickEditorParams,
+    handleExpiredSession: Boolean,
+    onAvatarSelected: () -> Unit,
```

Comment:
One thing we decided to do on iOS is to have a general `update()` callback (we call it `updateHandler()`, and it has a parameter with the information of which kind of update it is.

https://github.com/Automattic/Gravatar-SDK-iOS/blob/57d6110c3671f0da74480680e6fcb5012637be5e/Sources/GravatarUI/SwiftUI/ProfileEditor/QuickEditor.swift#L59

https://github.com/Automattic/Gravatar-SDK-iOS/blob/57d6110c3671f0da74480680e6fcb5012637be5e/Sources/GravatarUI/SwiftUI/ProfileEditor/QuickEditor.swift#L23-L35

Maybe is not needed yet but it will possible become handy when we start to implement the About Info editor.

Reply:
Thanks! The API design is yet to be decided on, so I will definitely look at iOS.